mtags/src/main/scala/scala/meta/internal/mtags/CharArrayReader.scala

```diff
@@ -61,16 +61,18 @@ private[mtags] case class CharArrayReader private (
     } else {
       begCharOffset = endCharOffset
       val (hi, hiEnd) = readUnicodeChar(endCharOffset)
+
       if (!Character.isHighSurrogate(hi)) {
         ch = hi
         endCharOffset = hiEnd
       } else if (hiEnd >= buf.length)
         readerError("invalid unicode surrogate pair", at = begCharOffset)
       else {
         val (lo, loEnd) = readUnicodeChar(hiEnd)
-        if (!Character.isLowSurrogate(lo))
-          readerError("invalid unicode surrogate pair", at = begCharOffset)
-        else {
+        if (!Character.isLowSurrogate(lo)) {
```

Comment:
This started failing on some of the sources and I have no idea why. It's not really important as it's only used for indexing.